lintro/ai/sanitize.py

```
@@ -0,0 +1,166 @@
+    Args:
+        content: Raw code content to sanitize.
+
+    Returns:
+        Sanitized content safe for prompt insertion.
+    """
+    if not content:
+        return content
+
+    result = _neutralize_role_markers(content)
+    result = _neutralize_xml_tags(result)
+    return result
```

Ruff (0.15.6)

[warning] Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)